com.unity.shadergraph/Editor/Generation/Processors/Generator.cs

```diff
@@ -649,17 +729,12 @@ void GenerateShaderPass(int targetIndex, PassDescriptor pass, ActiveFields activ
             // --------------------------------------------------
             // Dots Instanced Graph Properties
 
-            bool hasDotsProperties = false;
-            m_GraphData.ForeachHLSLProperty(h =>
-            {
-                if (h.declaration == HLSLDeclaration.HybridPerInstance)
-                    hasDotsProperties = true;
-            });
+            bool hasDotsProperties = subShaderProperties.HasDotsPropertiesForTarget(target);
```

Comment:
Moved this into the PropertyCollector, to keep it more consistent, and allow re-use of the HLSLProperty list.  Previously, it only considered Hybrid properties declared by the graph, but now it will consider them coming from any source.  While I don't think we have any Hybrid properties that can be declared by a Node or a Target, we could have them in the future.